doc/api/embedding.md

```diff
@@ -166,8 +166,47 @@ int RunNodeInstance(MultiIsolatePlatform* platform,
 }
 ```
 
+## Node-API Embedding
+
+<!--introduced_in=REPLACEME-->
+
+As an alternative, an embedded Node.js can also be fully controlled through
+Node-API. This API supports both C and C++ through [node-addon-api][].
```

Comment:
```suggestion
Node-API. This API supports both C and C++ through [node-addon-api][]. Although
the embedding API is not promised to be ABI stable at this time, it uses node-api types
and implementation so that it might be a some time in the future.
```